app/src/main/java/com/bogdan/codeforceswatcher/network/RestClient.kt

```diff
@@ -5,12 +5,18 @@ import com.google.gson.GsonBuilder
 import okhttp3.OkHttpClient
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
+import okhttp3.logging.HttpLoggingInterceptor
 
 object RestClient {
 
     private val retrofit by lazy {
         val okHttpClient = OkHttpClient.Builder()
-            .build()
+        val httpLoggingInterceptor = HttpLoggingInterceptor()
```

Comment:
This code can be improved in few ways:
1. Add interceptor only for debug builds (`BuildConfig.DEBUG)` (so we aren't spamming with network logs in production).
2. Use `apply` directly on `HttpLoggingInterceptor`, like this:

```
okHttpClient.addInterceptor(HttpLoggingInterceptor().apply { level = HttpLoggingInterceptor.Level.BODY })
```